Assets/MixedRealityToolkit.Examples/Demos/Gltf/Scripts/TestGlbLoading.cs

```diff
@@ -13,6 +13,7 @@ public class TestGlbLoading : MonoBehaviour
         [SerializeField]
         private string uri = string.Empty;
 
+#if UNITY_EDITOR
```

Comment:
These scripts can still work outside of the editor, it's just that the paths need to point to a valid path.

Reply:
Do you know if these files get picked up by default when building the mrtk? I am somewhat naive to this, we could add the files to resources but then i'm pretty sure that apps will pick them up independent of whether or not they are building this scene.

Reply:
FWIW, we are on a time crunch to get this in by friday. Getting both this pull request and your other pull request into mrtk_development outweigh fixing this demo to work outside the editor

Reply:
They work outside of the editor, it's just the resource paths need to point to something valid in a final built solution.

Reply:
Mainly the assets would have to be put into a streaming assets folder or accessed / downloaded locally via url, then that local path would be used to load the objects. 

Please revert the editor compiler symbols